docs/deprecated.md

```diff
@@ -18,7 +18,7 @@ keywords: "docker, documentation, about, technology, deprecate"
 This page provides an overview of features that are deprecated in Engine. Changes
 in packaging, and supported (Linux) distributions are not included. To learn
 about end of support for Linux distributions, refer to the
-[release notes](https://docs.docker.com/engine/release-notes/).
+[release notes](https://docs.docker.com/engine/release-notes/23.0/).
```

Comment:
slightly curious though; didn't we add a redirect for this? Or doesn't html proofer understand that one?

Reply:
! I thought that HTMLproofer doesn't understand the redirects - but I just noticed that my redirect was wrong. I have fixed it downstream now, checking to see if it works, in which case we can close this PR

Reply:
seems to work now, sorry for the noise, closing